loader.tf

```diff
@@ -36,3 +36,41 @@ resource "aws_s3_bucket_notification" "notifications" {
     filter_prefix       = "full"
   }
 }
+
+resource "aws_sns_topic" "success_sns_topic" {
+  name = var.success_topic_name
+
+  policy = <<POLICY
```

Comment:
Elsewhere we're using `iam_policy_document` resources which allow for using native terraform syntax to describe IAM policies instead of inline JSON blobs. https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html

Reply:
Ah, thank you. I've updated it to use `iam_policy_document` instead.